### PR TITLE
Add bzu tech documentation

### PR DESCRIPTION
**Pull request by enioo22**
## Proposed change

Add documentation to new core integration Bzu Tech.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107749
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5080
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bzutech.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ha_release: 2024.1
+ha_config_flow: true
+ha_iot_class: Cloud Polling
+ha_domain: bzutech
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The **Bzu Tech** {% term integration %} allows you to integrate your [Bzu Cloud](https://canary.is) sensors in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/bzutech.markdown]:
```suggestion
The **Bzu Tech** {% term integration %} allows you to integrate your [Bzu Cloud](https://canary.is) sensors in Home Assistant.
```
Do I need an account and login credentials as a prerequisite?

Comment (enioo22) [reply source/_integrations/bzutech.markdown]: > Do I need an account and login credentials as a prerequisite?

Yes

Comment (c0ffeeca7) [reply source/_integrations/bzutech.markdown]: would this be worth mentioning? as a prerequisite?

**Review by klaasnicolaas — COMMENTED** [review]
What can users expect from the integration? What can it do and what kind of entities do you get? (do not list the entity names, but describe them globally)

File: source/_integrations/bzutech.markdown (hunk 1 above)
Comment (enioo22) [reply source/_integrations/bzutech.markdown]: > would this be worth mentioning? as a prerequisite?

added having an account with registered devices as prerequisites

**Comment by enioo22** [comment]
> What can users expect from the integration? What can it do and what kind of entities do you get? (do not list the entity names, but describe them globally)

added a brief description of the entity (sensor) generated by the integration and some of its uses.

File: source/_integrations/bzutech.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ha_codeowners:
+  - '@enioo22'
+ha_domain: bzutech
+ha_quality_scale: platinum
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The **Bzu Tech** {% term integration %} allows you to integrate your [BzuCloud](https://cloud.bzutech.com.br) sensors and devices* in Home Assistant, generating a **Sensor {% term Entity %}** that can be used in automations, dashboards and more.
```
Comment (c0ffeeca7) [inline source/_integrations/bzutech.markdown]:
```suggestion
The **Bzu Tech** {% term integration %} allows you to integrate your [BzuCloud](https://cloud.bzutech.com.br) sensors and devices in Home Assistant. It provides a sensor {% term entity %} that can be used in automations, dashboards and more.
```

File: source/_integrations/bzutech.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+{% include integrations/config_flow.md %}
+
+*To use this integration you need a BzuCloud account with registered devices.
```
Comment (c0ffeeca7) [inline source/_integrations/bzutech.markdown]:
```suggestion

## Prerequisites

To use this integration, you need a BzuCloud account with registered devices.

{% include integrations/config_flow.md %}

```

**Review by c0ffeeca7 — COMMENTED** [review]
Style tweaks. Moved the Cloud requirement before the config flow because this is something the user should be aware of before installing the integration.

File: source/_integrations/bzutech.markdown (hunk 3 above)
Comment (c0ffeeca7) [reply source/_integrations/bzutech.markdown]: is there anything you can say about the type of sensors? Humidity, temperature, ...?

Comment (enioo22) [reply source/_integrations/bzutech.markdown]: > is there anything you can say about the type of sensors? Humidity, temperature, ...?

the integration is compatible with every type of bzu sensors: humidity, temperature, luminosity, co, co2, volatile organic compounds, voltage and current, do you think i should add it to the documentation??

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**